_posts/1970-01-01-admin.md

```diff
@@ -4,7 +4,8 @@ title: Admin
 order: 160
 ---
 
-Prefer solutions that will quiclky generate Admin area instead fo ActiveAdmin-like DSL that makes admin are quickly, but barely customizable
+There are multiple solutions that provide adding an administration panel to your Ruby app; instead of using an ActiveAdmin-like DSL.
+These solutions can help you easily create an admin panel although they're not so customizable.
```

Comment:
Not really clear what solutions do you mean by "these" - ActiveAdmin or other. Rephrase please to make direct emphasize on difficulties with ActiveAdmin customization.